tidb-cloud/v8.5-performance-highlights.md

```diff
@@ -0,0 +1,191 @@
+
+### Solution
+
+TiDB v8.5 introduces multiple enhancements to address the impact of cloud disk IO jitter on performance. These improvements effectively mitigate the effects of IO latency fluctuations, ensuring more stable and reliable performance.
+
+### Test environment
+
+- Cluster topology: TiDB (32 vCPU, 64 GiB) \* 3 + TiKV (32 vCPU, 64 GiB) \* 6
+- Workload: a read/write ratio of 2:1, with simulated cloud disk IO delays or hangs on one of TiKV nodes
+
+### Test results
+
+The failover time of the IO latency jitter is 30% shorter, and P99/999 latency is reduced by 70% or higher.
+
+- Test results without IO latency jitter improvement
+
+    | Workload | Failover time | QPS drop rate | Maximum latency (P999) | Maximum latency (P99) |
+    | --- |  --- |  --- |  --- |  --- |
+    | IO delay 1 s lasts for 10 mins | 3 mins | 93% | 4.66 s | 929 ms |
+    | IO delay 500 ms lasts for 10 mins | 2 mins | 92% | 7.22 s | 894 ms |
+    | IO delay 100 ms lasts for 10 mins | 3 mins | 80% | 7.53 s | 1.7 s |
+    | IO delay 50 ms lasts for 10 mins | 3 mins | 53% | 1.36 s | 238 ms |
+    | IO delay 10 ms lasts for 10 mins | 3 mins | 18% | 69 ms | 25 ms |
+    | IO delay 5 ms lasts for 10 mins | 2 mins | 29% | 37.9 ms | 10 ms |
+    | IO delay 2 ms lasts for 10 mins | Almost no effect | 1% | 14 ms | 7.9 ms |
+
+- Test results with IO latency jitter improvement
+
+    | Workload | Failover time | QPS drop rate | Maximum latency (P999) | Maximum latency (P99) |
+    | --- |  --- |  --- |  --- |  --- |
+    | IO delay 1 s lasts for 10 mins | 3 mins | 93% | 4.66 s | 929 ms |
+    | IO delay 500 ms lasts for 10 mins | 2 mins | 92% | 7.22 s | 894 ms |
+    | IO delay 100 ms lasts for 10 mins | 3 mins | 80% | 7.53 s | 1.7 s |
+    | IO delay 50 ms lasts for 10 mins | 3 mins | 53% | 1.36 s | 238 ms |
+    | IO delay 10 ms lasts for 10 mins | 3 mins | 18% | 69 ms | 25 ms |
+    | IO delay 5 ms lasts for 10 mins | 2 mins | 29% | 37.9 ms | 10 ms |
+    | IO delay 2 ms lasts for 10 mins | Almost no effect | 1% | 14 ms | 7.9 ms |
+
+## Batch processing
+
+### Challenge
+
+Large-scale transactions, such as bulk data updates, system migrations, and ETL workflows, involve processing millions of rows and are vital for supporting critical operations. While TiDB excels as a distributed SQL database, handling such transactions at scale presents two significant challenges:
+
+- Memory limits: in releases earlier than TiDB 8.1, all transaction mutations are held in memory throughout the transaction lifecycle, which strains resources and reduces performance. For operations involving millions of rows, this could lead to excessive memory usage and, in some cases, Out of Memory (OOM) errors when resources are insufficient.
```

Comment:
```suggestion
- Memory limits: in versions earlier than TiDB v8.1.0, all transaction mutations are held in memory throughout the transaction lifecycle, which strains resources and reduces performance. For operations involving millions of rows, this could lead to excessive memory usage and, in some cases, Out of Memory (OOM) errors when resources are insufficient.
```